wordpress/wp-content/plugins/cds-security-headers/cds-security-headers.php

```diff
@@ -87,11 +89,16 @@ function cds_security_headers($headers)
         }
     }
 
-    $headers['Content-Security-Policy'] = $csp;
+    if (! is_admin()) {
+        $headers['Content-Security-Policy'] = $csp;
+    }
+
+    $headers['strict-transport-security'] = 'max-age=31536000; includeSubDomains; preload';
+    $headers['X-XSS-Protection'] = '1; mode=block';
+    $headers['X-Frame-Options'] = 'SAMEORIGIN';
+    $headers['X-Content-Type-Options'] = 'nosniff';
 
     return $headers;
 }
 
-if (! is_admin()) {
-    add_filter('wp_headers', 'cds_security_headers');
-}
+add_filter('wp_headers', 'cds_security_headers');
```

Comment:
This only seems to run in the frontend (not in WP admin screens), so the if condition above isn't needed.